### DISCUSSION TITLE
Rendering issue

### DISCUSSION THREAD

**Discussion by nt7** [requarks/wiki#3974] [Error / Bug Report] [answered]
Hi,

I recently updated to v 2.5.201 running on Docker.

Today, i wanted to make a change to an existing page using the markdown editor: 
I see the chnages being displayed on the right preview pane, then I hit save, the rendering icon pops up and when I leave the edit mode, the changes are not displayed on the actual page.

The corresponding .md file save doin the server contains all the new changes, but they are not properly displayed/rendered?
If I re-render the whole wiki in the admin console, nothing changes.
If I create a new page, everything renders fine, even if I update this new page.
I checked the docker log and found:
`Loading configuration from /wiki/config.yml...
Rendering page ID 28...
Using database driver mysql2 for mariadb [ OK ]
Rendering page ID 28: [ FAILED ]
Cannot read property 'type' of undefined`

What could cause these issues? Caching issue?
Any ideas for a fix?

**Comment by winkelement** [comment]
I'm having the exact same issue.

**Reply by winkelement** [reply]
For me what fixed it was removing some leftover HTML comments (`<!-- -->`) in the page source, like

```
<!--
## Something

bla.

![img](/img/manuals/04_bla/bla_06.png){.elevation-3 .radius-5}
-->
```

**Reply by navinisaac** [reply]
That sorted my issue too..removed all HTML-style comments. Thanks!

**Comment by navinisaac** [comment]
Same issue here. It shows as an edit in history but refuses to render as live page.

**Comment by NGPixel** [comment, marked as answer]
See https://github.com/Requarks/wiki/issues/3863#issuecomment-852631212

**Reply by nickmonad** [reply]
I believe we are hitting a similar issue here, but this GitHub Issues link no longer resolves as it seems Issues have been removed in favor of Discussions.
